bundle/edu.gemini.pot/src/test/scala/edu/gemini/pot/sp/TestInstrumentCompleteness.scala

```diff
@@ -0,0 +1,26 @@
+package edu.gemini.pot.sp
+
+import edu.gemini.pot.sp.SPComponentBroadType.INSTRUMENT;
+
+import org.junit.Assert._
+import org.junit.Test
+
+/**
+ * Tests that the Instrument enumeration agrees with SPComponentType
+ */
+final class TestInstrumentCompleteness {
+
+  @Test
+  def testInstrumentEnumCompleteness(): Unit = {
+    val all = SPComponentType.values.toSet.filter(t => t.broadType == INSTRUMENT) -
+                SPComponentType.QPT_CANOPUS -
+                SPComponentType.QPT_PWFS
```

Comment:
These are kind of annoying and seem to have been introduced by mistake.  It would be good to get rid of them.  They aren't actually instruments or program nodes of any kind.